src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -301,6 +304,7 @@ public void getQueuesCount(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 100 to 0